stable/fluentd/templates/service.yaml

```diff
@@ -30,6 +30,12 @@ spec:
       targetPort: metrics
       protocol: TCP
   {{- end }}
+  {{- if eq .Values.service.sessionAffinity "ClientIP" }}
+  sessionAffinity: {{ .Values.service.sessionAffinity }}
```

Comment:
I would have instead a `None` as default value in values.yaml for a property `sessionAffinity` and include the if `ClientIP` below for sessionAffinityConfig.

Reply:
That is how it is done in other charts.

Reply:
Thanks for the tip, I've fixed the if condition. `sessionAffinity`  defaults to `None` in `values.yaml` so we are good here.